GeneratorInterface/Pythia8Interface/src/Py8HMC3InterfaceBase.cc

```diff
@@ -27,7 +27,11 @@ namespace gen {
 
     if (ps.exists("useEvtGenPlugin")) {
       useEvtGen = true;
-      string evtgenpath(std::getenv("EVTGENDATA"));
+      auto env = std::getenv("EVTGENDATA");
+      if (not env) {
+        cms::Exception("EvtGenMissingEnv") << "The environment variable EVTGENDATA must be defined";
```

Comment:
Should this `throw` ?

Reply:
Yes, thanks for the catch! (Pun unintended).